src/Polly.DependencyInjection/Polly.DependencyInjection.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net7.0</TargetFrameworks>
+    <UsePollyTarget>true</UsePollyTarget>
+    <AssemblyTitle>Polly.DependencyInjection</AssemblyTitle>
+  </PropertyGroup>
+
+
+  <ItemGroup>
+    <ProjectReference Include="..\Polly.Core\Polly.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Options" Version="7.0.0" />
+  </ItemGroup>
```

Comment:
+1 TFM and multi-targeting.

Reply:
Dropped this project. Not needed for POC.